core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -658,6 +658,7 @@ private static void objcopy(String... args) {
             static final Version SNAPSHOT_MANDREL = new Version("Snapshot", Integer.MAX_VALUE, Integer.MAX_VALUE,
                     Distribution.MANDREL);
             static final Version VERSION_20_1 = new Version("GraalVM 20.1", 20, 1, Distribution.ORACLE);
+            // Keeping 20.1 until Mandrel is updated
             static final Version CURRENT = VERSION_20_1;
```

Comment:
Yeah so that's where I think we need some tweaking because doing that, we will recommend using 20.1 in the log message if people are using an older version which is not what we want to do.

I think we need two constants: the minimal version and the current one.

Reply:
Btw, I wouldn't mind if I didn't think this would happen at each upgrade. There will always be a delay between Mandrel and GraalVM and we might release a version where both versions are not aligned.

Reply:
We can't release a final Mandrel 20.2 yet because we're depending on Java 11.0.9 changes coming up. GraalVM control their JDK (labs JDK) and they added some changes to the underlying JDK that is used for GraalVM 20.2. We're playing catch up here, unable to release final until 11.0.9 comes out.

I think we should maybe rename `CURRENT` to `MINIMUM`, to signal that's the minimum version that we support. That's all `CURRENT` is used for, to detect which is the minimum version.

Reply:
Having said that, we plan to release a Mandrel 20.2 beta shortly.

Reply:
No current is also used in the log message. So we need both MINIMUM and CURRENT.